rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -42,9 +42,46 @@ maas_target_alias: public0_v4
 maas_monitor_cinder_backup: "{{ cinder_service_backup_program_enabled | default(false) }}"
 
 # MaaS pathing and versions
-maas_release: master
+maas_release: "1.0.0"
```

Comment:
do we need to pin in master? it's not a release or anywhere near releasing, right?

Reply:
Sure, but it could require a release candidate at any time - so I was thinking that instead of only then pinning (delaying an RC) we should pin and sha bump just like any other development branch we have. ie This branch should follow the same release/update process.

Reply:
good for me. better safe than sorry.